.github/workflows/run_unit_tests.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest]
-        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12", "pypy3.8", "pypy3.9", "pypy3.10"]
```

Comment:
pypy3.9 on Windows takes 24 minutes to install packages and then fails with no output other than "Chef build error".
```
24m 46s
Run poetry install
Installing dependencies from lock file

Package operations: 44 installs, [1](https://github.com/ni/nidaqmx-python/actions/runs/10291434282/job/28483714820?pr=625#step:8:1) update, 0 removals

  - Installing markupsafe (2.1.5)
  - Installing mccabe (0.7.0)
  - Installing mypy-extensions (1.0.0)
  - Installing pathspec (0.12.1)
  - Installing platformdirs (4.2.2)
  - Installing pycodestyle (2.9.1)
  - Installing pyflakes (2.5.0)
  - Installing snowballstemmer (2.2.0)
  - Installing tomli (2.0.1)
  - Installing typing-extensions (4.12.2)
  - Installing black (24.4.2)
  - Installing distlib (0.3.8)
  - Installing exceptiongroup (1.2.1)
  - Installing filelock (3.15.1)
  - Installing flake8 (5.0.4)
  - Installing iniconfig (2.0.0)
  - Installing pluggy (1.5.0)
  - Installing pydocstyle (6.3.0)
  - Downgrading setuptools (70.1.0 -> 70.0.0)
  - Installing cachetools (5.3.3)
  - Installing chardet (5.2.0)
  - Installing coverage (7.5.3)
  - Installing flake8-black (0.3.6)
  - Installing flake8-docstrings (1.7.0)
  - Installing flake8-import-order (0.18.2)
  - Installing grpcio (1.64.1)
  - Installing isort (5.13.2)
  - Installing pep8-naming (0.14.1)
  - Installing protobuf (4.25.3)
  - Installing pyproject-api (1.6.1)
  - Installing pytest (8.2.2)
  - Installing toml (0.[10](https://github.com/ni/nidaqmx-python/actions/runs/10291434282/job/28483714820?pr=625#step:8:11).2)
  - Installing types-protobuf (4.25.0.20240417)
  - Installing virtualenv (20.26.2)

  ChefBuildError
```